articles/iot-edge/tutorial-nested-iot-edge.md

```diff
@@ -197,6 +218,24 @@ Install IoT Edge by following these steps on both devices.
    sudo dpkg -i ./libiothsm-std.deb
    sudo dpkg -i ./iotedge.deb
    ```
+1. Update package lists on your device.
+
+   ```bash
+   sudo apt-get update
+   ```
+
+1. Install the Moby engine.
+
+   ```bash
+   sudo apt-get install moby-engine
+   ```
+### Configure Virtual Machine
+If you use azure Virtual Machine to create a hierarchy of IoT Edge devices, make sure the the following ports are opened inbound: 8000,443,5671,8883.
+8000: Is used to pull docker container images through API proxy
+443: Is used between parent and child edgehub for Rest API calls.
+5671,8883: Are used for AMQP and MQTT
+
+[How to open ports to a virtual machine with the Azure portal](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/nsg-quickstart-portal).
```

Comment:
Can this go somewhere else in the tutorial, closer to the prerequisites where you create the virtual machines? It splits the flow here, falling between installing the runtime and provisioning it.

Reply:
Moved it up